index.html

```diff
@@ -665,17 +679,22 @@
           <li>Return <var>acceptPromise</var> and perform the remaining steps
           <a>in parallel</a>.
           </li>
-          <li>For each <var>paymentMethod</var> in
-          <var>request</var>.<a>[[\serializedMethodData]]</a>:
-            <ol>
-              <li>Determine which <a>payment apps</a> support any of the
-              <a>payment method identifiers</a> given by the first element of
-              the <var>paymentMethod</var> tuple. For each resulting payment
-              app, if payment method specific capabilities supplied by the
-              payment app match those provided by the second element of the
-              tuple, the payment app matches.
-              </li>
-            </ol>
+          <li><p><a>Extension point</a>: process any proprietary and/or other
+            supported algorithms, for determining the available methods of paying,
+            at this point in the algorithm.</p>
+            <p>Alternatively, the following generic algorithm may be applied:</p>
```

Comment:
There can't be an "alternatively". Otherwise it reads as:
```
if (extension point) {
  // 
} else{
  // alternatively
}
```

What we want is:

```
try {
  async for (const extension  of extensions) {
    await extension.doYourThing();
  }  
} catch(err){
   return reject(err);
}
// do stuff that is already in the spec
// this spec always wins, because extensions points must not override default behavior 
```